code/modules/modular_computers/file_system/programs/secureye.dm

```diff
@@ -4,7 +4,7 @@
 	filename = "secureye"
 	filedesc = "SecurEye"
 	category = PROGRAM_CATEGORY_MISC
-	program_icon_state = "generic"
+	program_icon_state = "camera"
```

Comment:
using the same icon as the camera app is a bit misleading.

Also, there is already a SecurEye icon for desktop modpcs that could be adapted (from the monitor console)